PetAdoption-iOS/PetAdoptionTransportKit/PTKRequestManager.swift

```diff
@@ -20,6 +23,14 @@ public typealias PTKRequestSpeciesComplete = (_ species: [PTKSpecieType]?, _ err
 public typealias PTKRequestPetsComplete = (_ pets: [PTKPet]?, _ error: NSError?) -> Void
 public typealias PTKRequestImageComplete = (_ image: UIImage?, _ error: NSError?) -> Void
 
+// PetFinder API
+private let PFBaseURL = "http://api.petfinder.com/"
+private let PFPetFind = "\(PFBaseURL)pet.find"
+
+private let API_KEY = "ca74980f2ab686e10a43095a87d24d45"
```

Comment:
Can we hide the API key?

Reply:
The main reason the API key is not hidden is really just because I'm not the one that's going to be pushing this to the App Store.  I could put it into some file and put that file into the .gitignore, but then when the app gets pushed, that file might get left out, causing the app to not work.  I'm open to suggestions on how we can hide the key and also ensure that it doesn't get forgotten when it comes time to deploy.